.gitignore

```diff
@@ -1,4 +1,5 @@
 compiled
+doc
```

Comment:
Why ignore `doc`?

Reply:
Because it's automatically generated by `raco setup` when I use the repo directory as a linked package.